c/GraphQLAstVisitor.cpp

```diff
@@ -50,6 +50,6 @@ void graphql_node_visit(const struct GraphQLAstNode *node,
 {
   CVisitorBridge visitor(callbacks, userData);
   if (node) {
-    reinterpret_cast<const facebook::graphql::ast::Node *>(node)->accept(&visitor);
+    reinterpret_cast<const graphql::ast::Node *>(node)->accept(&visitor);
```

Comment:
@leebyron Was this by accident? I assume it is breaking. Its related to your larger question about the namespace at https://github.com/graphql/libgraphqlparser/issues/94

Reply:
Actually, this breaks the build process:

```
libgraphqlparser/c/GraphQLAstVisitor.cpp:53:28: error: 
      use of undeclared identifier 'graphql'; did you mean 'facebook::graphql'?
    reinterpret_cast<const graphql::ast::Node *>(node)->accept(&visitor);
                           ^~~~~~~
                           facebook::graphql
```

I will fix.

Reply:
Fixed in #95.